tensorflow/contrib/lite/examples/ios/simple/AppDelegate.m

```diff
@@ -14,20 +14,12 @@
 
 #import "AppDelegate.h"
 
-#import "RunModelViewController.h"
+//#import "RunModelViewController.h"
```

Comment:
Remove this line? In fact since you aren't doing anything in the delegate how about we get rid of this file altogether and just pass nil to UIApplicationMain

Reply:
There must be an object implementing app delegate methods in every iOS application, without it the app wouldn’t launch properly. It’s easier and more convenient to keep a class implementing those methods in separate files.